src/controllers/actionItemController.spec.js

```diff
@@ -0,0 +1,117 @@
+const actionItemController = require('./actionItemController');
+const {
+  mockReq,
+  mockRes,
+  mongoHelper: { dbConnect, dbDisconnect },
```

Comment:
We can delete this import, since we are not using the memory database in any unit test.